src/mscorlib/shared/System/Threading/ThreadAbortException.cs

```diff
@@ -18,13 +18,21 @@
 
 namespace System.Threading
 {
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public sealed class ThreadAbortException : SystemException
     {
         internal ThreadAbortException()
+            : base(GetMessageFromNativeResources(ExceptionMessageKind.ThreadAbort))
```

Comment:
Why do you need to do this?

This change will not compile in CoreRT. If you have a good reason to do this, you will need to ifdef this out.

Reply:
it was a difference between the generated blob in netfx and core. I think I will revert this change and refine the equality comparison for ThreadAbortException in the test code.